add-on/chrome.manifest

```diff
@@ -1,24 +0,0 @@
-content loop chrome/content/ contentaccessible=yes
-locale loop en-US chrome/locale/en-US/
```

Comment:
Please can you add a line to jar.mn to replace this. I'd forgotten about it before. You should just be able to add:

`% locale loop en-US %locale/en-US/`

If you could put that under the content line in add-on/jar.mn that'd be great